salt/elasticsearch/files/config/readonlyrest-docs.yml

```diff
@@ -1,14 +1,12 @@
 readonlyrest:
   # https://github.com/beshu-tech/readonlyrest-docs/blob/master/kibana.md#loading-settings-order-of-precedence
   force_load_from_file: True
-  ssl:
-    server_certificate_key_file: {{ pillar.elasticsearch.plugins.readonlyrest.certificate_key_file }}
-    server_certificate_file: {{ pillar.elasticsearch.plugins.readonlyrest.certificate_file }}
   # https://github.com/beshu-tech/readonlyrest-docs/blob/master/actionstrings/action_strings_es7.10.1.txt
   access_control_rules:
-    - name: Allow localhost
-      hosts:
-        - localhost
+    - name: Allow manage admin access over localhost
```

Comment:
@jpmckinney, The original "Allow localhost" configuration doesn't work because all connections appear as localhost. 
I've implemented this work around that gives the "manage" user more access when they are querying directly from on ocp19.

I was trying to make a dedicated "ocpadmin" user for this but no matter what I tried ES wouldn't recognise my new user. 
How did you make the current "public" and "manage" users?
What tool did you use to generate passwords?

Reply:
How did you generate the `auth_key_sha512` value?

Reply:
echo -n 'username:password' | shasum -a 512

Reply:
`ocpadmin` user created and working fine.